src/postprocess/band_structure.jl

```diff
@@ -30,3 +30,20 @@ function compute_bands(ham::Hamiltonian, kpoints, n_bands;
 
     select_eigenpairs_all_kblocks(band_data, 1:n_bands)
 end
+
+function plot_bands(ham, n_bands, kline_density, composition, εF)
+    basis = ham.basis
+    # Band structure calculation along high-symmetry path
+    kpoints, klabels, kpath = high_symmetry_kpath(basis, kline_density, composition...)
+    println("Computing bands along kpath:\n     $(join(kpath[1], " -> "))")
+    band_data = compute_bands(ham, kpoints, n_bands)
+
+    # Plot bandstructure using pymatgen
+    plotter = pyimport("pymatgen.electronic_structure.plotter")
+    bs = pymatgen_bandstructure(basis, band_data, klabels, fermi_level=εF)
+    bsplot = plotter.BSPlotter(bs)
+    plt = bsplot.get_plot()
```

Comment:
Don't you think this should go to `src/external/pymatgen.jl` instead? I think it's wise to keep the explicit pymatgen-dependent code to one place inside DFTK.

Reply:
We can move it out later if you want. That's a postprocessing function, as far as I'm concerned it can go either as API in pymatgen (in which case we need to refactor it a bit) or postprocessing

Reply:
I'm really not too sure, the `high_sym_kpath` function also depends on it, so one should split the API there as well. It's just that pymatgen really is a big beast and things such as the `ZeroDivisionError` really start to annoy me a bit, so I'd like to keep the option open to easily change at some later point.

I can agree to your definition of post-processing function, but then make a quick note about that in the readme (i.e. these are just stand-alone tools, potentially depending on further third-party code and not part of DFTK, something like that)